einstein-ai/1.0/modules/ROOT/pages/einstein-ai-connector-reference.adoc

```diff
@@ -0,0 +1,660 @@
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  | 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+=== Throws
+
+* MS-EINSTEIN:CONNECTIVITY 
+* MS-EINSTEIN:EMBEDDING_OPERATIONS_FAILURE 
+* MS-EINSTEIN:RAG_FAILURE 
+* MS-EINSTEIN:RETRY_EXHAUSTED 
+* MS-EINSTEIN:TOOLS_OPERATION_FAILURE 
+
+
+[[RagAdhocLoadDocument]]
+== RAG Adhoc Load Document
+`<ms-einstein:rag-adhoc-load-document>`
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Prompt a| String | Plain text prompt to send to the LLM. |  #[payload] | 
+| File Path a| String | Full file path for the document to ingest into the embedding store. Ensure the file path is accessible. |  | x
+| Output Mime Type a| String | MIME type of the payload that this operation outputs. |  | 
+| Output Encoding a| String | Encoding of the payload that this operation outputs. |  | 
+| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  | 
+| Embedding Name a| String |  |  sfdc_ai__DefaultOpenAITextEmbeddingAda_002 | 
+| File Type a| String a| Type of the document to ingest into the embedding store:
+
+* Text
+* PDF
+* CSV
+* URL  |  PDF | 
+| Option Type a| String | How to split the document prior to ingestion into the embedding or vector database. |  FULL | 
+| Model API Name a| String | Name of the API model that interacts with the LLM. |  sfdc_ai__DefaultOpenAIGPT35Turbo | 
+| Probability a| Number |Probability of the model API staying accurate.  |  0.8 | 
+| Locale a| String | Localization information, which can include the default locale, input locale(s), and expected output locale(s). |  en_US | 
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  | 
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  #[payload] | 
+| Error Mappings a| Array of <<ErrorMapping>> |  Set of error mappings |  | 
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  | 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| <<EinsteinResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+=== Throws
+
+* MS-EINSTEIN:CONNECTIVITY 
+* MS-EINSTEIN:EMBEDDING_OPERATIONS_FAILURE 
+* MS-EINSTEIN:RAG_FAILURE 
+* MS-EINSTEIN:RETRY_EXHAUSTED 
+* MS-EINSTEIN:TOOLS_OPERATION_FAILURE 
+
+
+[[ToolsUseAiService]]
+== Tools Use Ai Service
```

Comment:
was this operation in Einstein AI Connector?

Reply:
Yes